### DISCUSSION TITLE
Tree House

### DISCUSSION THREAD

**Discussion by sfwhat** [sunflower-land/sunflower-land#1196] [Art]
### Tree House Artwork

This could be a Carpenter NPC's house or an upgraded version of our home.

I don't want things to get more complicated until land expansion but this was my art proposal on Aha! and I did not wanted it to get lost. 

**_Proof of demand_**
![image](https://user-images.githubusercontent.com/95937808/177793020-44045553-e203-43ca-860a-3a06f5cd1e08.png)
___

#### Preview
![treehouse (1)](https://user-images.githubusercontent.com/95937808/177792743-30ca8207-a0a9-46e1-b0da-96899680b114.gif)

Asset ![treehouse (1)](https://user-images.githubusercontent.com/95937808/177792754-729e8bbd-6965-4cb4-9fa5-ea929584a8c2.png)
Layered aseprite file: [treehouselayeredwithman2.zip](https://github.com/sunflower-land/sunflower-land/files/9064442/treehouselayeredwithman2.zip)

**Comment by MrSnorch** [comment]
It looks beautiful 😍
